bench/client.go

```diff
@@ -57,6 +74,23 @@ func (h *HasClient) InitIndex(index string, frame string) error {
 	return h.client.SyncSchema(h.schema)
 }
 
+func (h *HasClient) InitRange(index, frame, field string, maxValue, minValue int64) error {
```

Comment:
I'm being picky here, but can we swap `minValue` and `maxValue` so they are in ascending order?